src/server/lock_manager/waiter_manager.rs

```diff
@@ -551,6 +588,11 @@ impl WaiterManager {
         cb(self.wait_table.borrow().to_wait_for_entries());
     }
 
+    fn handle_dump_history(&self, cb: Callback) {
+        let history = mem::take(self.wait_table.borrow_mut().history.as_mut());
```

Comment:
Just `take` out the `wait_table` but not clone?

Reply:
I planned to use ngm to pull this, and clean the in memory records after pulled.
Maybe provide a seperated clean API is better.